.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+	"workbench.colorCustomizations": {
+		"activityBar.background": "#063601",
+		"titleBar.activeBackground": "#084B02",
+		"titleBar.activeForeground": "#EEFFEC"
+	}
+}
```

Comment:
@twisttaan @apoorvcodes this file is irrelevant

Reply:
already removed it